### ISSUE TITLE
Putting small buy and sell order 

### ISSUE THREAD

**Issue by zillabytes** [BuildOnViction/bug-reports#36]
Is this normal to put small amount on buy and sell order? but if we are maker or taker there is a minimum for trade previously i posted on other issue. here is the example of my screenshot putting small sell order of XRP.
![makertaker](https://user-images.githubusercontent.com/38444647/72491582-74badb80-3855-11ea-831a-4b984bcf945d.jpg)

**Comment by zillabytes** [comment]
here's an additional sell order on the price of 11 XRP.
![makertaker1](https://user-images.githubusercontent.com/38444647/72493723-b8651380-385c-11ea-852f-bb9f710c24f0.jpg)

**Comment by thanhson1085** [comment]
For now, you can create a small amount order, but it will be rejected by order-matching engine, then the status of the orders will be updated to "Rejected"
What is your expectation? you want to validate the minimum of the amount before creating the orders?

**Comment by zillabytes** [comment]
Yes i think there's should be a validation amount so that it will not put on the section of sell order, it can lead to spam the sell order or buy order, and if some one play the network it will appear replacing the real buy/sell order. Just like what I did.

**Comment by thanhson1085** [comment]
Ok, we should validate the mininum.

Thank you. Confirmed @chinhnt23

**Comment by chinhnt23** [comment]
Thank you @thanhson1085. Confirmation received 
@zillabytes Could you send me your Wallet Address pls ?

**Comment by zillabytes** [comment]
0x9ca510605e82ff444d80b4cf32a56280b3d7f117

Tnx..

**Comment by chinhnt23** [comment]
@zillabytes  check your wallet please, thank you!

**Comment by zillabytes** [comment]
got it sir.. thank you
